server/GlideScriptedExtensionPoint.ts

```diff
@@ -0,0 +1,10 @@
+
+declare class GlideScriptedExtensionPoint {
+/**
+   * Get extension point by name.
+   * @param extensionName The name of the extension point.
+   */
+  getExtensions(extensionName: string): Array<any>;
```

Comment:
I would suggest to make this a generic method. Using `any` is not recommended. It will also provide IntelliSense and type-checking when using extension points.

Also, the preferred way of defining arrays is using square brackets.

```suggestion
  getExtensions<TExtension>(extensionName: string): TExtension[];
```

Reply:
Thanks. Should be resolved. @bbarber9  you can check and merge eventually.